src/Components/App/App.js

```diff
@@ -1,21 +1,39 @@
 import React, { Component } from 'react';
+import movieData from '../../movieData'
 import Header from '../Header/Header';
 import Movies from '../Movies/Movies';
-import movieData from '../../movieData'
+import Details from '../Details/Details'
 
 class App extends Component {
   constructor() {
     super()
     this.state = {
-      allMovies: movieData.movies
+      allMovies: movieData.movies,
+      isMovieSelected: false,
+      selectedMovieID: null,
     }
   }
 
+  toggleSeletion = (movieID) => {
+    this.setState({
+      isMovieSelected: !this.state.isMovieSelected, 
+      selectedMovieID: movieID,
+     })
+  }
+
   render() {
     return (
       <>
         <Header />
-        <Movies allMovies={this.state.allMovies}/>
+        {!this.state.isMovieSelected && 
```

Comment:
Nice conditional rendering! It can be helpful to bump lines 29-31 in one more tab, and to put the closing } on it's own line, for readability.